libraries/AP_Motors/AP_MotorsHeli_RSC.h

```diff
@@ -5,6 +5,22 @@
 #include <RC_Channel/RC_Channel.h>
 #include <SRV_Channel/SRV_Channel.h>
 
+// default main rotor speed (ch8 out) as a number from 0 ~ 1000
+#define AP_MOTORS_HELI_RSC_SETPOINT             700
+
+// Throttle Curve Defaults
+#define AP_MOTORS_HELI_RSC_THRCRV_0_DEFAULT     250
```

Comment:
Peter Barker and others would say that we should be moving to use enum classes instead of defines because it adds extra type safety.  It's not a blocker but in the future moving to enums is a good idea.